party-role/src/main/java/org/fiware/tmforum/partyrole/domain/AccountRef.java

```diff
@@ -0,0 +1,32 @@
+package org.fiware.tmforum.partyrole.domain;
+
+import java.net.URI;
+import java.util.List;
+
+import org.fiware.tmforum.common.domain.RefEntity;
+
+import io.github.wistefan.mapping.annotations.AttributeGetter;
+import io.github.wistefan.mapping.annotations.AttributeSetter;
+import io.github.wistefan.mapping.annotations.AttributeType;
+import io.github.wistefan.mapping.annotations.Ignore;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
```

Comment:
AccountRef, CreditProfile and PaymentMethodRef already exist as part of the customer-management module. It should be moved to  a shared module and reused